tests/iscsi/iscsi_client.pm

```diff
@@ -16,50 +16,145 @@ use strict;
 use warnings;
 use testapi;
 use mm_network;
+use mm_tests;
 use lockapi;
+use version_utils qw(is_sle is_leap);
+use yast2_widget_utils 'change_service_configuration';
 use x11utils 'turn_off_gnome_screensaver';
-use y2_module_consoletest;
+use utils qw(systemctl type_string_slow_extended);
+use scheduler 'get_test_data';
 
-sub run {
+# load expected test data from yaml
+# common for both iscsi MM modules
+my $test_data = get_test_data();
+
+sub prepare_xterm_and_network {
     x11_start_program('xterm -geometry 160x45+5+5', target_match => 'xterm');
```

Comment:
I know that this code was there before, but these are exactly same steps as we have in `iscsi_server` module, and I guess worth extracting.